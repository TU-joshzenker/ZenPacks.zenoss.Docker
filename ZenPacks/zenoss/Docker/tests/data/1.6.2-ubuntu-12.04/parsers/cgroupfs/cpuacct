
/sys/fs/cgroup/cpuacct/

/sys/fs/cgroup/cpuacct/docker

/sys/fs/cgroup/cpuacct/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa

/sys/fs/cgroup/cpuacct/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/cpuacct.stat
user 1
system 1

/sys/fs/cgroup/cpuacct/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/cpuacct.usage_percpu
27113768 20524563 

/sys/fs/cgroup/cpuacct/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/cpuacct.usage
47638331

/sys/fs/cgroup/cpuacct/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/notify_on_release
0

/sys/fs/cgroup/cpuacct/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/tasks
1648
1667

/sys/fs/cgroup/cpuacct/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/cgroup.clone_children
0

/sys/fs/cgroup/cpuacct/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/cgroup.event_control

/sys/fs/cgroup/cpuacct/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/cgroup.procs
1648
1667

/sys/fs/cgroup/cpuacct/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f

/sys/fs/cgroup/cpuacct/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/cpuacct.stat
user 1484824
system 689290

/sys/fs/cgroup/cpuacct/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/cpuacct.usage_percpu
10694197656820 10863578271913 

/sys/fs/cgroup/cpuacct/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/cpuacct.usage
21557782399617

/sys/fs/cgroup/cpuacct/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/notify_on_release
0

/sys/fs/cgroup/cpuacct/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/tasks
1587
1611
1612
1613
1614
1615
1616
1617
1618
1619
1620
1621
1622
1623
1624

/sys/fs/cgroup/cpuacct/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/cgroup.clone_children
0

/sys/fs/cgroup/cpuacct/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/cgroup.event_control

/sys/fs/cgroup/cpuacct/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/cgroup.procs
1587
1611
1612
1613
1614
1615
1616
1617
1618
1619
1620
1621
1622
1623
1624

/sys/fs/cgroup/cpuacct/docker/cpuacct.stat
user 1484828
system 689293

/sys/fs/cgroup/cpuacct/docker/cpuacct.usage_percpu
10694256227444 10863635438883 

/sys/fs/cgroup/cpuacct/docker/cpuacct.usage
21557897311728

/sys/fs/cgroup/cpuacct/docker/notify_on_release
0

/sys/fs/cgroup/cpuacct/docker/tasks

/sys/fs/cgroup/cpuacct/docker/cgroup.clone_children
0

/sys/fs/cgroup/cpuacct/docker/cgroup.event_control

/sys/fs/cgroup/cpuacct/docker/cgroup.procs

/sys/fs/cgroup/cpuacct/cpuacct.stat
user 1485451
system 692939

/sys/fs/cgroup/cpuacct/cpuacct.usage_percpu
10894226699490 10891736570521 

/sys/fs/cgroup/cpuacct/cpuacct.usage
21785966943247

/sys/fs/cgroup/cpuacct/release_agent


/sys/fs/cgroup/cpuacct/notify_on_release
0

/sys/fs/cgroup/cpuacct/tasks
1
2
3
5
7
8
9
10
11
12
13
14
15
16
17
19
20
21
22
23
24
25
26
27
28
29
30
31
32
33
34
37
38
39
40
41
42
43
55
57
76
77
205
206
360
364
371
380
387
429
430
522
594
811
825
863
874
875
876
901
908
918
919
922
933
934
942
969
972
973
990
1090
1091
1195
1199
1200
1201
1202
1204
1206
1260
1360
1361
1465
1469
1470
1544
1557
1561
1574
1575
1584
1625
1629
1636
1637
1638
1639
1643
1644
1645
1646
1647
1653
1666
2414
3113
3620
3917
3950
3968
3972
4006

/sys/fs/cgroup/cpuacct/cgroup.sane_behavior
0

/sys/fs/cgroup/cpuacct/cgroup.clone_children
0

/sys/fs/cgroup/cpuacct/cgroup.event_control

/sys/fs/cgroup/cpuacct/cgroup.procs
1
2
3
5
7
8
9
10
11
12
13
14
15
16
17
19
20
21
22
23
24
25
26
27
28
29
30
31
32
33
34
37
38
39
40
41
42
43
55
57
76
77
205
206
360
364
371
380
387
429
430
522
594
811
825
863
901
908
918
919
922
933
934
942
969
972
990
1090
1091
1195
1199
1260
1360
1361
1465
1469
1470
1575
1584
1636
1643
2414
3113
3620
3950
3968
3972
4010
