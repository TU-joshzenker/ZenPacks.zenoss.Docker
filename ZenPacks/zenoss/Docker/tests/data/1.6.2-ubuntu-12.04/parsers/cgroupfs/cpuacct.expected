# vim: ft=yaml
command: '/usr/bin/env sudo find /sys/fs/cgroup/cpuacct/ -printf \"\n%p\n" -exec cat {} 2>/dev/null \;'

values:
  9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f:
    usage: 1077889
    usage_user: 742412
    usage_system: 344645
    processes: 16

